lfs-image

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+set +e
+
+# Config options you may pass via Docker like so 'docker run -e "<option>=<value>"':
+# - IMAGE_NAME=<name>, define a static name for your .bin files
+
+cd /opt/nodemcu-firmware
+
+if [ ! -x luac.cross -a !  -x luac.cross.int ]; then
+  echo Error: No cross compiler found. You need to build the firmeware first.
+  exit -1
+fi
+
+export BUILD_DATE
+BUILD_DATE="$(date "+%Y-%m-%d %H:%M")"
+LUA_FILES=$(find ../lua -iname "*.lua")
+echo Adding files: ${LUA_FILES}
+
+if [ -z "$IMAGE_NAME" ]; then
+  IMAGE_NAME=LFS_"${BUILD_DATE}"
+fi
+
+# make a float LFS image if available
+if [ -x luac.cross ]; then
+  echo creating "${IMAGE_NAME}".bin
+  find ../lua -iname "*.lua" -exec ./luac.cross -f -o ../lua/"${IMAGE_NAME}".bin {} +
```

Comment:
Did you (i.e. Terry and you) recently change the file ending convention from `.img` to `.bin`?

Reply:
Good catch. This should be img.